platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -441,6 +444,16 @@ public boolean isDestroyed() {
     return destroyed;
   }
 
+  /**
+   * Set the strict mode that will throw the {@link com.mapbox.mapboxsdk.MapStrictModeException}
+   * whenever the map would fail silently otherwise.
+   *
+   * @param strictModeEnabled true to enable the strict mode, false otherwise
+   */
+  public void setMapStrictModeEnabled(boolean strictModeEnabled) {
+    MapStrictMode.setStrictModeEnabled(strictModeEnabled);
+  }
```

Comment:
it feels strange exposing a static configuration as non static on MapView. For example, you enable it for one Mapview and it will be continued to be enabled for others (eventhough other codepath don't touch this method). Imo every new instance of MapView should behave the same. I feel we either makes this method static as well or we remove it as the configuration is already publicly available.

Reply:
Good point. I added this method to the `MapView` solely to improve discoverability. Making it static should be the way to go I think.